prodtests/check_embedding_pileup.sh

```diff
@@ -0,0 +1,51 @@
+# Script verifying pileup / embedding features of digitizers
+# It is a basic and fully automatic status check of what should
+# work
+
+# The approach is the following:
+# 1) for each detector we simulate 2 events that both leave hits
+# 2) we digitize the events with the same collision time assigned to both of them
+#    and check whether the output digits have multiple labels --> checks pileup
+# 3) we digitize with trivial embedding: background and signal events are the same
+#    and check whether the output digits have multiple labels --> checks embedding
+
+# o2-sim -j 4 -g pythia8 -n 4 --seed 1 ok for 
+# CVP, MCH, MID, HMP need special gun
+
+dets=(       ITS     TPC   TRD    EMC     PHS    TOF      CPV    HMP     MCH   MID   MFT   FV0   FT0   FDD   ZDC )
+generators=(boxgen boxgen boxgen boxgen hmpidgun  boxgen  boxgen  hmpidgun fwpigen fwpigen fwpigen  fddgen fddgen fddgen pythia8 )
+
+for idx in "${!dets[@]}"; do
+  d=${dets[$idx]}
+  gen=${generators[$idx]}
+  # we put the detector plus pipe and magnet as materials
+  o2-sim-serial -m PIPE MAG ${d} -g ${gen} -n 2 --seed 1 -o o2sim${d} > simlog${d} 2>&1
+
+  unlink o2sim.root
+  unlink o2sim_grp.root
+  ln -s o2sim${d}.root o2sim.root
+  ln -s o2sim${d}_grp.root o2sim_grp.root
+
+  # verify that we have hits
+  NUMHITS=`root -q -b -l ~/alisw_new/O2/macro/analyzeHits.C | grep "${d}" | awk '//{print $2}'`
+  echo "Found ${NUMHITS} hits for ${d}" 
+  
+  # digitize with extreme bunch crossing
+  o2-sim-digitizer-workflow --onlyDet ${d} -b --tpc-lanes 1 > digilog${d} 2>&1
```

Comment:
Very nice! But here one should invoke with an option ``o2-sim-digitizer-workflow --interactionRate 1e8 --onlyDet ...`` (due to the numerical problems the higher rates require the [fix](https://github.com/AliceO2Group/AliceO2/pull/2816) I've just posted)).

Reply:
Yes. This script is just a basic draft and needs to be finalized to be useful in automatic checks. (I still like to add some engineered sequence of hits to be absolutely sure that pileup should trigger). For the moment this PR provides the more important `analyzeLabelDigits` macro which should already work to check for embedding.